### PR TITLE
Update Dockerfile

### PR DESCRIPTION
**Pull request by nporteschaikin**
- Use standard node image (includes git)

### CODE REVIEW

**Comment by nporteschaikin** [comment]
Tested this - it builds and runs.

**Comment by GordonDiggs** [comment]
What was the node version on the old image? What does this do to the size of the image?

File: Dockerfile [hunk 1]
```diff
@@ -1,8 +1,8 @@
-FROM alpine:edge
+FROM mhart/alpine-node:6.10.1
 
-RUN apk --update add nodejs git
 WORKDIR /root/
 
+RUN apk --update add git
```
Comment (GordonDiggs) [inline Dockerfile]: You can get a nice size bump if you add this into the next line:

```
RUN apk --update add git && \
      git clone https://github.com/etsy/statsd.git && \
      cd statsd && git reset --hard v0.7.2 && \
      apk del git && \
     npm install statsd-librato-backend statsd-datadog-backend
```

**Comment by nporteschaikin** [comment]
Glad you asked. Using `node` was a mistake; it's massive (>700 MB). Moved to a tagged version of alpine. 20 MB on `master`, 77.5 MB here.

**Comment by GordonDiggs** [comment]
Nice

**Comment by nporteschaikin** [comment]
60 MB now.

File: Dockerfile (hunk 1 above)
Comment (nporteschaikin) [inline Dockerfile]: 👍

**Pull request merged**